homeassistant/components/airly/air_quality.py

```diff
@@ -68,11 +70,32 @@ def __init__(self, airly, name, unique_id):
         self._icon = "mdi:blur"
         self._attrs = {}
 
+    async def async_added_to_hass(self):
+        """Call when entity is added to HA."""
+        self._async_unsub_dispatcher_connect = async_dispatcher_connect(
+            self.hass, TOPIC_DATA_UPDATE, self._update_callback
+        )
+        self._update_callback()
+
+    async def async_will_remove_from_hass(self):
+        """Disconnect dispatcher listener when removed."""
+        if self._async_unsub_dispatcher_connect:
+            self._async_unsub_dispatcher_connect()
+        # pylint: disable=protected-access
```

Comment:
Is this work around required here?

Reply:
Without disabled `protected-access` there are pylint warnings W0212 for lines 85, 86 and 87.